Gems/Atom/RHI/Vulkan/Code/Source/RHI/Device.cpp

```diff
@@ -1128,7 +1128,15 @@ namespace AZ
                 && m_enabledDeviceFeatures.sparseResidencyImage3D
                 && m_enabledDeviceFeatures.sparseResidencyAliased
                 && deviceProperties.sparseProperties.residencyStandard2DBlockShape
-                && deviceProperties.sparseProperties.residencyStandard3DBlockShape;
+                && deviceProperties.sparseProperties.residencyStandard3DBlockShape && ([this]{
```

Comment:
Add a quick comment about why this code was added.

Reply:
// check if there is a queue family that supports both VK_QUEUE_TRANSFER_BIT and VK_QUEUE_SPARSE_BINDING_BIT.